src/lib/objects/split/views/settle-now.svelte

```diff
@@ -29,18 +29,31 @@
 	export let exitObject: () => void
 	export let send: (message: DataMessage) => Promise<void>
 
-	let owedAmount = 0n
+	let owedAmount: undefined | bigint = undefined
 	let settling = false
-	$: {
-		const balance = balances.find(({ address }) => address === profile.address)
-		if (balance) {
-			owedAmount = BigInt(balance.amount)
-		}
-	}
+	let fee: bigint | undefined = undefined
+	let feeError: undefined | Error = undefined
```

Comment:
`feeError` is not set anywhere. Maybe on line 51 it could be set?

Reply:
good catch! Forgot to commit this one, thanks!